src/main/java/org/opensearch/knn/plugin/rest/RestGetModelHandler.java

```diff
@@ -50,7 +50,7 @@ public List<Route> routes() {
     @Override
     protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient client) throws IOException {
         String modelID = restRequest.param(MODEL_ID);
-        if (!Strings.hasText(modelID)) {
+        if (!StringUtils.isBlank(modelID)) {
```

Comment:
if we are replacing Strings.hasTest, we should replace it with StringUtils.isNotBlank

Reply:
updated